lighthouse-extension/gulpfile.js

```diff
@@ -198,26 +188,15 @@ gulp.task('browserify-lighthouse', () => {
     .pipe(gulp.dest('dist/scripts'));
 });
 
-gulp.task('browserify-other', () => {
+gulp.task('js-other', () => {
   return gulp.src([
     'app/src/popup.js',
-  ], {read: false})
-    .pipe(tap(file => {
-      let bundle = browserify(file.path); // , {debug: true}); // for sourcemaps
-      bundle = applyBrowserifyTransforms(bundle);
-
-      // Inject the new browserified contents back into our gulp pipeline
-      file.contents = bundle.bundle();
-    }))
```

Comment:
`popup.js` doesn't `require()` any other files and hasn't in a long time, so no need to run it through browserify.